modules/joda/src/test/scala/pureconfig/module/joda/configurable/ConfigurableSuite.scala

```diff
@@ -74,6 +73,48 @@ class ConfigurableSuite extends FlatSpec with Matchers with EitherValues with Pr
       case class Conf(yearMonth: YearMonth)
       conf.to[Conf].right.value shouldEqual Conf(yearMonth)
   }
+
+  it should "parse Instant" in forAll {
```

Comment:
Nitpick: please move the tests for types which are not configurable to a `pureconfig.module.joda.JodaSpec` file.